dev-python/gitana/gitana-0.2.ebuild

```diff
@@ -0,0 +1,77 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit distutils-r1 eutils
+
+DESCRIPTION="SQL-based Git Repository Inspector"
+HOMEPAGE="https://github.com/SOM-Research/Gitana"
+SRC_URI="https://github.com/SOM-Research/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+IUSE=""
+
+KEYWORDS="~amd64 ~x86"
+
+# problem with https://github.com/SOM-Research/Gitana/issues/5
+# wait for release of git-python version 2.0.9 and update as soon as possible
+
+DEPEND="dev-lang/python:2.7[tk]
```

Comment:
hmmm, strange... as soon as I change that line into `dev-lang/python:2.7[tk,${PYTHON_USEDEP}]` I get the following output by repoman:
```bash
dependency.bad [fatal]        44
   dev-python/gitana/gitana-0.2.ebuild: DEPEND: ~amd64(default/linux/amd64/13.0)
[     'dev-lang/python:2.7[tk,python_targets_python2_7(-)?,-python_single_target_python2_7(-)]']
```